ee/clickhouse/queries/trends/util.py

```diff
@@ -73,9 +73,7 @@ def get_active_user_params(filter: Union[Filter, PathFilter], entity: Entity, te
     diff = timedelta(days=7) if entity.math == WEEKLY_ACTIVE else timedelta(days=30)
     if filter.date_from:
         params.update(
-            {
-                "parsed_date_from_prev_range": f"AND timestamp >= '{format_ch_timestamp(filter.date_from - diff, filter)}'"
-            }
+            {"parsed_date_from_prev_range": f"AND timestamp >= '{format_ch_timestamp(filter.date_from - diff)}'"}
```

Comment:
Is this supposed to need the timezone arg?

Reply:
it explicitly does not. I made the decision that when we pass through those people URLs all of the dates should be in UTC for consistency. In general when working with timezones you really only want toto use timezones for user display, not for moving times around. see https://github.com/PostHog/posthog/pull/9678/files/98f36e23df1acb96128c75cdd01bb6d8a1837290#diff-cf067174aadfdcda82472c96d1e1278a8b8884f3ec17a1869b0e0e26ce586552R145